test/fakeintake/api/api.go

```diff
@@ -8,3 +8,12 @@ package api
 type APIFakeIntakePayloadsGETResponse struct {
 	Payloads [][]byte `json:"payloads"`
 }
+
+type RouteStat struct {
```

Comment:
It's not particularly easy to work with this structure. We're sure the `ID` is unique, so why do we need an array?

Reply:
Updated !